README.md

```diff
@@ -49,6 +49,23 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [official TiDB blog](https://www.pingcap.com/blog/).
 
+
+## Quick start
+
+### To start using TiDB
+
+Read the [Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/ ), which includes deployment methods using TiUP, Ansible, Docker, and Kubernetes.
+
+### To start developing TiDB
```

Comment:
Contribute to TiDB

Reply:
Ditto, referred from k8s https://github.com/kubernetes/kubernetes#to-start-developing-kubernetes